packages/babel-preset-react-app/index.js

```diff
@@ -87,6 +87,21 @@ if (env === 'test') {
     plugins: plugins
   };
```

Comment:
While you're at it, maybe we can remove the special case on lines 69-74? Seems like https://github.com/babel/babel/pull/5088 fixed it. As a separate PR.